applications/nrf_desktop/doc/selector.rst

```diff
@@ -0,0 +1,53 @@
+.. _selector:
+
+selector module
+###############
+
+The ``selector`` module is used to send event (``selector_event``) informing about current selector state.
+Module has one implementation, which uses hardware selectors (``selector_hw.c``).
+
+Module Events
+*************
+
++----------------+-------------+-----------------+--------------------+------------------+
+| Source Module  | Input Event | This Module     | Output Event       | Sink Module      |
++================+=============+=================+====================+==================+
+|                |             | ``selector``    | ``selector_event`` | :ref:`ble_bond`  |
++----------------+-------------+-----------------+--------------------+------------------+
+
+Configuration
+*************
+
+Hardware selectors
+==================
+
+Module uses the Zephyr :ref:`zephyr:gpio` driver to check state of the hardware selectors - it is required to set :option:`CONFIG_GPIO` option.
+
+In addition set ``CONFIG_DESKTOP_SELECTOR_HW_ENABLE`` and define module configuration.
+The configuration (array of :c:type:`struct selector_config` pointers) is written in the ``selector_hw_def.h`` file located in the board specific directory in application configuration folder.
+
+For every hardware selector you should define:
+* ID (:cpp:member:`id`),
+* pointer to array of :c:type:`struct gpio_pin` (:cpp:member:`pins`) and size of the array (:cpp:member:`pins_size`).
+
+.. warning::
+
+  Please make sure that each source of ``selector_event`` has unique ID.
+  Otherwise it won't be possible to properly distinguish events from different sources.
+
+Implementation details
+**********************
+
+Hardware selectors
+==================
+
+Module sends ``selector_event`` informing about current hardware selector state:
+* on system start or wake up (for every defined selector),
+* when selector state changes, while application is running (only selector, which changed state).
+
+When application goes to sleep interrupts are disabled (selectors are not informing about state change).
+
+If a selector is placed in between states no event is send (state of this selector is unknown).
```

Comment:
```suggestion
If a selector is placed between states, no event is sent (state of this selector is unknown).
```
Same question as for line 48: What does the information in parentheses mean? It is the reason why the interrupts are not sent? Or the information of the consequence of what happens after they are not sent? Clarify in the text.

Reply:
done